arch/xtensa/src/esp32s3/hardware/esp32s3_systimer.h

```diff
@@ -0,0 +1,808 @@
+/****************************************************************************
+ * arch/xtensa/src/esp32s3/hardware/esp32s3_systimer.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_SYSTIMER_H
+#define __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_SYSTIMER_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include "esp32s3_soc.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* SYSTIMER_CONF_REG register
+ * Configure system timer clock
+ */
+
+#define SYSTIMER_CONF_REG (DR_REG_SYSTIMER_BASE + 0x0)
+
+/* SYSTIMER_CLK_EN : R/W; bitpos: [31]; default: 0;
+ * register file clk gating
+ */
+
+#define SYSTIMER_CLK_EN    (BIT(31))
```

Comment:
minor
```suggestion
#define SYSTIMER_CLK_EN    BIT(31)
```
here and in other places